src/colvarbias_abf.cpp

```diff
@@ -832,3 +835,43 @@ int colvarbias_abf::write_output_files()
   write_gradients_samples(output_prefix);
   return COLVARS_OK;
 }
+
+cvm::real colvarbias_abf::calc_energy()
+{
+  if (num_variables() != 1) return 0.0;
+  
+  // Get the home bin.
+  int home0 = gradients->current_bin_scalar(0);
+  if (home0 < 0) return 0.0;
+  int home = (home0 < gradients->number_of_points(0))?home0:gradients->number_of_points(0)-1;
+ 
+  // Integrate the gradient up to the home bin.
+  double sum = 0.0;
```

Comment:
It would be best to use `cvm::real`. We probably won't ever use single-precision math for production, but this is not guaranteed.

Reply:
I'll change this to cvm::real.